library/public/window.h

```diff
@@ -76,6 +76,11 @@ class F3D_EXPORT window
    */
   virtual int getHeight() const = 0;
 
+  /**
+   * Set the position of the window
+   */
+  virtual window& setPosition(int x, int y) = 0;
```

Comment:
@Meakk

Reply:
Since we have `getWidth` and `getHeight`, I think it makes sense to have `getPositionX` and `getPositionY`, but I don't know if VTK implementations works.

Reply:
I'm unable to make it work, either before render, after a render or even during interaction using a timer. I'd say we drop it for now.

Reply:
Not tested in VTK at first glance: `grep -R  GetPosition | grep Window | grep Testing`